src/seedsigner/controller.py

```diff
@@ -466,3 +466,21 @@ def handle_exception(self, e) -> Destination:
             exception_msg,
         ]
         return Destination(UnhandledExceptionView, view_args={"error": error}, clear_history=True)
+
+    @property
+    def active_view(self) -> View:
```

Comment:
As mentioned in a previous comment, I would not expose this. 

I would just do this retrieval within the `can_run_screensaver()` method.